string app_name
string text
---
bool success
